docker/router/config.json

```diff
@@ -1,37 +1,28 @@
 {
-  "adminToken": "blahblah",
-  "chainConfig": {
-    "1337": {
-      "providers": ["http://172.17.0.1:8545"],
-      "confirmations": 1,
-      "subgraph": "http://172.17.0.1:8010/subgraphs/name/connext/nxtp",
-      "transactionManagerAddress": "0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0"
-    },
-    "1338": {
-      "providers": ["http://172.17.0.1:8546"],
-      "confirmations": 1,
-      "subgraph": "http://172.17.0.1:9010/subgraphs/name/connext/nxtp",
-      "transactionManagerAddress": "0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0"
-    }
+  "logLevel": "debug",
```

Comment:
I'm not sure this hardcoded config file needs to be in here for just the 2000/3000 domains (rinkeby, kovan). In the old stuff, this config file was used to spin up a fully local integration test environment. At some point, we will want to replicate that, but I think even then we can structure it better. IMO it shouldn't be inside this `docker/router/` directory/